src/ukify/ukify.py

```diff
@@ -1016,7 +1019,8 @@ def make_uki(opts):
 
         if names[0] != '.profile':
             raise ValueError(
-                f'Expected .profile section as first valid section in PE profile binary {profile} but got {names[0]}'
+                'Expected .profile section as first valid section in '
+                f'PE profile binary {profile} but got {names[0]}'
```

Comment:
No, let's not do this. Mixing non-f-strings and f-strings is error prone. This also breaks grepping for messages. Please keep the strings which are one logical line in one chunk, even if it exceeds the expected line length a bit.